features/step_definitions/view_restaurant_page_steps.rb

```diff
@@ -1,6 +1,9 @@
 Given("the following restaurants exist") do |table|
   table.hashes.each do |hash|
-    FactoryGirl.create(:restaurant, hash)
+    restaurant = FactoryGirl.create(:restaurant, hash)
+    binding.pry
+    @res_category = RestaurantCategory.find { |category| category.name == hash[:name]}
```

Comment:
.find(hash[:restaurant_category]. Also, no need for an instance variable. Local will do

Reply:
Will change this according to las tthree comments.  Thanks for the input